src/EditorFeatures/Core.Cocoa/NavigationCommandHandlers/FindBaseSymbolsCommandHandler.cs

```diff
@@ -63,9 +64,11 @@ protected override bool TryExecuteCommand(int caretPosition, Document document,
             {
                 using (var token = _asyncListener.BeginAsyncOperation(nameof(StreamingFindBaseSymbolsAsync)))
                 {
-                    // Let the presented know we're starting a search.
+                    // Let the presented know we're starting a search.  We pass in no cancellation token here as this
+                    // operation itself is fire-and-forget and the user won't cancel the operation through us (though
+                    // the window itself can cancel the operation if it is taken over for another find operation.
```

Comment:
💡 This behavior is more clear in the caller's context than it is here. Consider requiring a `CancellationToken` for the async method, and moving this comment and the use of `CancellationToken.None` up to the caller.